app/models/manageiq/providers/storage_manager/swift_manager/refresh_parser.rb

```diff
@@ -34,7 +34,7 @@ def ems_inv_to_hashes
     end
 
     def object_store
-      return if @swift_service.blank? || @swift_service.name != :swift
+      return if @swift_service.nil? || @swift_service.blank? || @swift_service.name != :swift
```

Comment:
Any reason for adding the extra nil check?  nil is blank, so it shouldn't be needed.

Reply:
Brain fart.  I think I thought about changing it, not adding it.  I'll revert that line...